substrate/frame/merkle-mountain-range/src/lib.rs

```diff
@@ -59,19 +59,16 @@
 extern crate alloc;
 
 use alloc::vec::Vec;
-use frame_support::weights::Weight;
-use frame_system::pallet_prelude::{BlockNumberFor, HeaderFor};
 use log;
-use sp_mmr_primitives::utils;
-use sp_runtime::{
-	traits::{self, One, Saturating},
-	SaturatedConversion,
+
+pub use frame::{
+	deps::sp_mmr_primitives::{
+		self as primitives, utils, utils::NodesUtils, Error, LeafDataProvider, LeafIndex, NodeIndex,
+	},
+	prelude::*,
```

Comment:
This should not be exported publicly

Reply:
https://github.com/paritytech/polkadot-sdk/pull/7081/commits/565985b655f6cfe4b879406ca0b7bef5897fac78